### PR TITLE
Eclde 6308 images monitoring chart

### PR DESCRIPTION
**Pull request by Jgrant-tw**

### CODE REVIEW

File: Lumen Edge Private Cloud/General/cpc-metrics.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{{
   "title": "CPC Metrics",
-  "date": "6-15-2020",
+  "date": "06-15-2020",
```
Comment (ldsz89) [inline Lumen Edge Private Cloud/General/cpc-metrics.md:3]: This date is incorrect.

**Comment by Jgrant-tw** [comment]
Please upload new version of cpc-metrics2.png to the dccf folder.

File: Lumen Edge Private Cloud/General/cpc-metrics.md (hunk 1 above)
Comment (Jgrant-tw) [reply Lumen Edge Private Cloud/General/cpc-metrics.md:3]: I may have put that date on, but that article is in a different branch. The date change is NOT SUPPOSED to be part of this pull request. I'll do a different pull request for the entire article once I do pull requests for all the images in the article.

Comment (Jgrant-tw) [reply Lumen Edge Private Cloud/General/cpc-metrics.md:3]: Sorry for the confusion -- please delete this pull request.

**Pull request closed without merging**